src/main/java/io/cdap/e2e/pages/actions/CdfHubActions.java

```diff
@@ -0,0 +1,127 @@
+/*
+ * Copyright © 2023 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package io.cdap.e2e.pages.actions;
+
+import io.cdap.e2e.pages.locators.CdfHubLocators;
+import io.cdap.e2e.pages.locators.CdfPluginPropertiesLocators;
+import io.cdap.e2e.utils.AssertionHelper;
+import io.cdap.e2e.utils.ElementHelper;
+import io.cdap.e2e.utils.PluginPropertyUtils;
+import io.cdap.e2e.utils.SeleniumHelper;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * CDF hub related actions.
+ */
+public class CdfHubActions {
+
+  private static final Logger logger = LoggerFactory.getLogger(CdfHubActions.class);
+  static {
+    SeleniumHelper.getPropertiesLocators(CdfHubLocators.class);
+  }
+
+  public static void clickOnHub() {
+    ElementHelper.clickOnElement(CdfHubLocators.hubPage);
+  }
+  public static void clickOnPlugin(String pluginName) {
+    ElementHelper.clickOnElement(CdfHubLocators.locatePlugin(pluginName));
+  }
+  public static void clickOnButton(String buttonName) {
+    ElementHelper.clickOnElement(CdfHubLocators.locateButton(buttonName));
+  }
+  public static void verifyPipelineStatus(String pipelineName) {
+    ElementHelper.isElementDisplayed(CdfHubLocators.locateButton(pipelineName));
+  }
+  public static void clickOnDeploy() {
+    ElementHelper.clickOnElement(CdfHubLocators.deployButton);
+  }
+  public static void clickOnFinish() {
```

Comment:
Provide some info in comment before starting of actions that what is doing . 
/**
   * Click on Finish button to deploy the plugin Or create the pipline .
   */
Please provide this type of info in comments for each and every actions

Reply:
done